src/nspawn/nspawn-mount.c

```diff
@@ -185,9 +187,11 @@ static int tmpfs_patch_options(
                 const char *selinux_apifs_context,
                 char **ret) {
 
-        char *buf = NULL;
+        char *buf = strdup(options);
+	if (!buf)
```

Comment:
also, another nitpick: we try to avoid to declare variables and invoke functions in one line. i.e. please write this as "char *buf;" and then "buf  = strdup(…);" on another line...